core/usecase/get-comments.usecase.go

```diff
@@ -0,0 +1,23 @@
+package usecase
+
+import (
+	"github.com/nhaancs/GrabGoTrainingWeek5Assignment/core/repository"
+)
+
+// GetCommentsUsecase usecase
+type GetCommentsUsecase struct {
```

Comment:
we should only expose interface, construct
implement struct should be left private